### DISCUSSION TITLE
Why to define QUALITY?

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#120] [Q&A]
Hi,
Why I need to define quality for my V3 cam, since despite what I define in quality environment setting, it will just take what defined in Wyze Ap.
Example, if Quality=HD120, and wyze cam app set as SD, so it will just stream via RTSP SD ...

**Comment by mrlt8** [comment]
You don't _need_ to define it. The QUALITY parameter was always optional.

Yes, unfortunately, the wyzecams seem to change the resolution of the streams with/without the app which can cause issues. I believe some of the dead streams that required a container restart were partly caused by getting stuck in a loop of having the wrong frame size.  Ignoring the frame size and bitrate would be fine for the RTSP stream, but, as we recently saw, the RTMP and HLS had some issues.

v0.6.4 should detect any change to the stream and restart the connection and force it back to the one set in the QUALITY and v0.6.5 will default to HD120 if QUALITY is not defined.

**Reply by AdiAbuAli** [reply]
Thanks.
